js/main.js

```diff
@@ -0,0 +1,26 @@
+const $imgInput = document.querySelector('#photo-url');
+const $imgSrc = document.querySelector('#img');
+
+function imgChange(event) {
+  $imgSrc.setAttribute('src', event.target.value);
+}
+
+$imgInput.addEventListener('input', imgChange);
+
+const $form = document.querySelector('form');
+
+function formStore(event) {
+  event.preventDefault();
+  const entry = {
+    title: $form.elements.title.value,
+    photoUrl: $imgSrc.getAttribute('src'),
```

Comment:
Works, but you should grab the `value` itself from the `$form.elements` just like you're doing with the other properties.